airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io.airbyte.cdk/read/CdcPartitionsCreatorFactory.kt

```diff
@@ -19,12 +19,6 @@ class CdcPartitionsCreatorFactory(
         val opaqueStateValue: OpaqueStateValue? = stateQuerier.current(feed)
         return when (feed) {
             is Global -> {
-                // TODO: remove the following if-block
-                if (opaqueStateValue != null) {
-                    // For the time being, only do one round
-                    return null
```

Comment:
returning a `null` here tells `FeedReader.createPartitions()` that this factory cannot handle this type of feed, which leads to a system error exception.

Reply:
Indeed